app/Http/Controllers/transactionController.php

```diff
@@ -83,4 +84,15 @@ public function destroy($id)
         $data->delete();
         return redirect()->route('listTrans');
     }
+    public function receipt ($id)
+    {
+        $data = tran::find($id);
+        $trans_data = [
```

Comment:
```suggestion
        $transaction = [
```
Use descriptive names & also only use camelCase for identifiers.

Reply:
Similar changes throughout.